src/main/java/com/epam/test_generator/entities/api/TokenTrait.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.test_generator.entities.api;
+
+import com.epam.test_generator.entities.Token;
+
+import java.util.Date;
+
+public interface TokenTrait {
+
+    Token getInstance();
```

Comment:
getInstance() should not be here. Instead, pull method  getExpiryDate.

Reply:
Done